tests/migrations/test_operations.py

```diff
@@ -1094,6 +1094,41 @@ def test_add_binaryfield(self):
         self.assertEqual(bytes(pony.digits), b"42")
         self.assertEqual(bytes(pony.quotes), b'"\'"')
 
+    def test_add_foreignkey(self):
+        """
+        Tests that adding a ForeignKey creates the corresponding constraint.
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things. Ticket references should be reserved for obscure issues (not needed here). These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).